packages/opentelemetry-core/test/common/ConsoleLogger.test.ts

```diff
@@ -114,5 +116,33 @@ describe('ConsoleLogger', () => {
       consoleLogger.debug('debug called %s', 'param1');
       assert.strictEqual(debugCalledArgs, null);
     });
+
+    if (typeof process !== 'undefined' && process.release.name === 'node')
```

Comment:
process is not available in browser so we shouldn't imho have test that is meant only for node that is run in browser too.

Reply:
true, I am not sure how to set the env in the test for both node and browser, that is why I just test for node with `        process.env.OTEL_LOG_LEVEL = ...;`, how can I set an env var for both platforms in the test?

Reply:
can you try to mock/overwrite getEnv or parseEnvironment so it will be platform unindependent ?